src/errors/HttpErrors.ts

```diff
@@ -0,0 +1,17 @@
+import BaseError from './BaseError'
+
+export class TaskNotFoundError extends BaseError {
+  constructor (status = 404, message = 'Task not found') {
+    super(status, message)
```

Comment:
I think, you could simply pass status and message to `super(400, '')`  and you don't need  to have them in constructor.

Reply:
In case when status and message do not have values , set in constructor , you need to set values when class instance is created . For example in task controller
![image](https://user-images.githubusercontent.com/50378914/134628758-11fbf9cf-7f35-4ab5-b5be-78169d089227.png)

Reply:
c612b1feb30118ac3c59790b282badd72380a0de

Reply:
![image](https://user-images.githubusercontent.com/4456550/134632984-c938b7f9-92fd-4d93-a875-24fb27957dc6.png)

Reply:
We pass status and message to ctor of BaseError, and we don't need them here.

Reply:
61484fc9f1f6b2093aedd924